### ISSUE TITLE
Mark `readOnlyRestToken` as secret

### ISSUE THREAD

**Issue by mattfysh** [upstash/pulumi-upstash#14]
Currently only `restToken` and `password` are marked as secret outputs, but I believe the read-only REST token should also be included. Cheers!

**Comment by praneetloke** [comment]
@mattfysh Pulumi supports specifying marking additional properties as secret outputs through the resource option [`additionalSecretOutputs`](https://www.pulumi.com/docs/concepts/options/additionalsecretoutputs/). Would that work for you as a workaround?

**Comment by mattfysh** [comment]
@praneetloke this is great, I'll use it instead. It would be nice to have it in the SDK so that it's not something you can forget to add, but this'll do for now. Cheers!

**Comment by praneetloke** [comment]
> It would be nice to have it in the SDK so that it's not something you can forget to add

@mattfysh agreed. I think you should reopen this issue to track that perhaps?

**Comment by mattfysh** [comment]
This would be fixed by https://github.com/upstash/terraform-provider-upstash/issues/37

**Comment by mattfysh** [comment]
Hi @praneetloke - the upstream terraform provider has merged the fix: https://github.com/upstash/terraform-provider-upstash/pull/39

Is it possible to pull through the updated via source, or do we need to wait for a versioned release to hit the terraform registry? thanks!

**Comment by praneetloke** [comment]
Hi @mattfysh, it's controlled by the version of the upstream provider that this provider depends on in [`go.mod`](https://github.com/upstash/pulumi-upstash/blob/master/provider/go.mod#L12). Someone from Upstash would need to update this provider to either point to an updated release of the TF provider (for which Upstash would have to do a new release) or update this to point to a specific commit in the TF provider repo since Go modules support [pseudo-versions](https://go.dev/ref/mod#pseudo-versions) by referencing a commit hash.

Either way, this provider would need to be updated so the updated version is available in all the relevant language registries (npm, PyPi etc.) The alternative of forking this repo leads to more work and undesirable in my opinion unless it's warranted due to some breaking issue (or if this provider is abandoned.)

cc @burak-upstash

**Comment by burak-upstash** [comment]
Hey @mattfysh and @praneetloke ! So sorry for the super late response here.
We are in the process of updating our terraform provider. We will shortly update our Pulumi provider after releasing the new version on Terraform side.
